algorithmic_efficiency/workloads/imagenet_resnet/imagenet_jax/workload.py

```diff
@@ -30,7 +30,9 @@ def _build_dataset(self,
                      data_dir: str,
                      global_batch_size: int,
                      cache: Optional[bool] = None,
-                     repeat_final_dataset: Optional[bool] = None):
+                     repeat_final_dataset: Optional[bool] = None,
+                     use_mixup: bool = False,
+                     mixup_alpha: float = 0.1):
```

Comment:
we can remove mixup_alpha from here, and then in the imagenet_vit base class, we have a 2 line `_build_dataset` function that just passes `use_mixup=(split == 'train')` to the super class' implementation (same for pytorch)